content/en/docs/examples/alb-nlb/index.md

```diff
@@ -0,0 +1,363 @@
+
+    {{< text bash >}}
+    $ export NLB_SECRET=<your NLB secret's name which appears as the last value in the output of the previous command>
+    {{< /text >}}
+
+1.  Extract the certificate and the key from the secret provided for NLB:
+
+    {{< text bash >}}
+    $ mkdir nlb_certs
+    $ kubectl get secret $NLB_SECRET --namespace=default -o yaml | grep 'tls.key:' | cut -f2 -d: | base64 --decode > nlb_certs/tls.key
+    $ kubectl get secret $NLB_SECRET --namespace=default -o yaml | grep 'tls.crt:' | cut -f2 -d: | base64 --decode > nlb_certs/tls.crt
+    $ ls -al nlb_certs
+    -rw-r--r--   1 user  staff  1679 Sep 11 07:55 tls.key
+    -rw-r--r--   1 user  staff  3921 Sep 11 07:55 trusted.crt
+    {{< /text >}}
+
+1.  Extract the certificate and the key from the secret provided for ALB:
+
+    {{< text bash >}}
+    $ mkdir alb_certs
+    $ kubectl get secret $ALB_SECRET --namespace=default -o yaml | grep 'tls.key:' | cut -f2 -d: | base64 --decode > alb_certs/client.key
+    $ kubectl get secret $ALB_SECRET --namespace=default -o yaml | grep 'tls.crt:' | cut -f2 -d: | base64 --decode > alb_certs/client.crt
+    $ ls -al alb_certs
+    -rw-r--r--   1 user  staff  3738 Sep 11 07:57 client.crt
+    -rw-r--r--   1 user  staff  1675 Sep 11 07:57 client.key
+    {{< /text >}}
+
+1.  Download the issuer certificate of the [Let's Encrypt](https://letsencrypt.org) certificate, which is the
+    issuer of the certificates provided by IBM Cloud Kubernetes Service.
+
+    {{< text bash >}}
+    $ curl https://letsencrypt.org/certs/trustid-x3-root.pem --output trustid-x3-root.pem
+    {{< /text >}}
+
+1.  Append the issuer certificate of [Let's Encrypt](https://letsencrypt.org) to the certificate of NLB
+    (currently required for ALB):
+
+    {{< text bash >}}
+    $ cat nlb_certs/tls.crt trustid-x3-root.pem > trusted.crt
+    {{< /text >}}
+
+1.  Create Kubernetes secrets to be used by Istio ingress gateway and ALB to establish mutual TLS between them. Note
+    that the name of the secrets for the Istio ingress gateway must be exactly as in the commands.
+
+    {{< warning >}}
+    The certificates provided by IBM Cloud Kubernetes Service expire every 90 days and are automatically renewed by
+    IBM Cloud Kubernetes Service 37 days before they expire.
+    You will have to recreate the secrets by rerunning the instructions of this section every time the secrets provided
+    by IBM Cloud Kubernetes Service are updated. You may want to use scripts or operators to automate this and keep the
+    secrets in sync.
+    {{< /warning >}}
+
+    {{< text bash >}}
+    $ kubectl create -n istio-system secret tls istio-ingressgateway-certs --key nlb_certs/tls.key --cert trusted.crt
+    $ kubectl create -n istio-system secret generic istio-ingressgateway-ca-certs --from-file=trustid-x3-root.pem
+    $ kubectl create secret generic alb-certs -n istio-system --from-file=trusted.crt --from-file=alb_certs/client.crt --from-file=alb_certs/client.key
```

Comment:
who is using alb-certs?

Reply:
The ALB.

Reply:
u mean istio ingress gateway?

Reply:
no, the ALB uses alb-certs to communicate with the Istio ingress gateway via mutual TLS

Reply:
Yeah, the issue is complex. Maybe the summary will clarify it, added in https://github.com/istio/istio.io/pull/5098/commits/e6307814cc8f1a640f6ad8461994f9573c018f46